apps/examples/tls_client/tls_client_main.c

```diff
@@ -462,7 +462,7 @@ int tls_client_cb(void *args)
 	int tail_len;
 	int i;
 	int written;
-	int frags;
+	int frags = 0;
```

Comment:
could you please mention, what build error.

Reply:
Uninitialized variable warning treated as error.